<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GauSei</class>
 <widget class="QMainWindow" name="GauSei">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GauSei</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="show2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Limpia">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="showi">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Carga">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargar Datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sale">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X1:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="Tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <row>
     <property name="text">
      <string>Ecuacion 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>=</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>252</red>
        <green>203</green>
        <blue>204</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>252</red>
        <green>203</green>
        <blue>204</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>252</red>
        <green>203</green>
        <blue>204</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>252</red>
        <green>203</green>
        <blue>204</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="imax">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iteraciones:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iteraciones:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>-10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ingrese los valores de cada ecuaci√≥n</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error Aceptado:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="errora">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-99999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="Inicia">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);</string>
    </property>
    <property name="text">
     <string>    Resultados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>591</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_11</zorder>
   <zorder>label_8</zorder>
   <zorder>show2</zorder>
   <zorder>Limpia</zorder>
   <zorder>showi</zorder>
   <zorder>Carga</zorder>
   <zorder>Sale</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>show1</zorder>
   <zorder>Tabla</zorder>
   <zorder>imax</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>show3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_6</zorder>
   <zorder>errora</zorder>
   <zorder>Inicia</zorder>
   <zorder>show4</zorder>
   <zorder>label_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Sale</sender>
   <signal>clicked()</signal>
   <receiver>GauSei</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
